TAC/Projects/Enarx/enarx-proposal.md

```diff
@@ -0,0 +1,66 @@
+# Enarx project contribution proposal 
+
+1. name of project
+   1. Enarx
+
+1. Project description (what it does, why it is valuable, origin and history)
+   1.  Enarx aims to make it simple to deploy workloads to a variety of different TEEs in the cloud, on your premises or elsewhere, and to allow you to have confidence that your application workload is as secure as possible.
+   1.  The problem we're trying to address is that there are many sensitive workloads that you shouldn't entrust to a public cloud to run, or may even have concerns about running on on-premises systems. TEEs (see below) provide a great opportunity to help secure these workloads, but they're not easy to use. Enarx aims to make it simple to deploy workloads to a variety of different TEE technologies in the cloud, on your premises or elsewhere, and to ensure that your application workload is as secure as possible.
+   1.  Enarx came out of some conversations between Nathaniel McCallum and Mike Bursell (the project’s founders) and led to them presenting the idea to a number of people at Red Hat, who agreed to provide some initial resources to see if it was plausible.  When the Consortium was announced, we realised very quickly that it was very well aligned.
+
+1. statement on alignment with Consortium charter's mission
+   1.  Enarx aims to be hardware-vendor neutral, providing a run-time environment which works across platforms.  We hope to make it easy for people to deploy their own applications or applications created by 3rd party vendors, supporting the “protecting data in use” part of the charter which is core to the Consortium.  Being conceived as an open source project, we welcome input and collaboration from all parties.  Minimal Trusted Compute Base and high auditability are high on our list of design principles. The code is very modular, meaning that there may well be opportunities for other projects to re-use components for their needs. 
+
+1. link to *current* Code of Conduct (if one is adopted already)
+   1.  TBD. 
+
+1. sponsor from TAC, if identified (a sponsor helps mentor projects)
+   1.  Mike Bursell (Red Hat)
+
+1. project license 
+   1.  Apache 2.0
+
+1. source control (GitHub by default)
+   1.  GitHub
+
+1. issue tracker (GitHub by default)
+   1.  GitHub
+
+1. external software dependencies (including licenses)
+   1.  TBD
```

Comment:
https://docs.google.com/spreadsheets/d/1UKnbbGWXYLjnPZsox3zmYo59nv3XSXjePfas5E2fER0/edit

Reply:
Updated and pushed.